.github/workflows/composite/dp-integ-tests/action.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: dp-integ-tests
+description: Run DP integration tests
+
+inputs:
+  make_tests_target:
+    required: true
+    description: Make target for running relevant tests
+  test_dir:
+    required: true
+    description: Directory to store tests results
+    default: /tmp/integration-tests-results
+  github_token:
+    required: true
+    description: Github token
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+    - name: Set env
+      shell: bash
+      run: |
+        echo "MINIKUBE_DP_MAX_MEMORY=$(grep MemTotal /proc/meminfo | awk '{printf "%dm",$2/1024 - 1}')" >> $GITHUB_ENV
```

Comment:
Watch out for consistent spacing between steps. The other ones are separated by a "free line".

Reply:
I aligned it